README.md

```diff
@@ -204,12 +245,12 @@ Requests/sec:  10710.55
 Transfer/sec:      1.81MB
 ```
 
-### UvReactor
+### Benchmark with UvReactor
 
 Start server:
 ```bash
 cd ./aerys
-php -n -dzend.assertions=-1 -dextension="/usr/local/opt/php70-uv/uv.so" ./vendor/bin/aerys -w 1 -c ./server.php
+php -n -dzend.assertions=-1 -dextension="/usr/local/opt/php70-uv/uv.so" ./vendor/bin/aerys -w 1 -c ./server.php  --worker-args="-n"
```

Comment:
@kelunik I'm passing extensions configuration using INI files and aerys awere about it and pass these files into workers as well:
https://github.com/amphp/aerys/blob/98bf472fc5f5c2b3f26f9f1daebaf2663b7f7066/lib/WatcherProcess.php#L391